src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -221,10 +221,14 @@ const CpsOnwardJourney = ({
         ) : null}
         {hasSingleContent ? (
           <SingleContentWrapper columnType={columnType}>
-            {promoComponent({ promo: singleContent, dir })}
```

Comment:
this component being called as a function was for some reason causing infinite loops for us when we implemented the view tracker

Reply:
hmm, I'd love to know what's going on with that 🤔

Reply:
Too tight on time to investigate but I think we should be using JSX rather than functions to render components